doc/manpages/qvm-features.rst

```diff
@@ -45,7 +45,96 @@ have some specific capability.
 Some extensions interpret the values as boolean. In this case, the empty string
 means :py:obj:`False` and non-empty string (commonly ``'1'``) means
 :py:obj:`True`. An absence of the feature means "default", which is
-extension-dependent.
+extension-dependent. In most cases the default value for feature is retrieved
+from a qube template.
+
+List of known features
```

Comment:
I'd like to see explicit "default: \<something\>" at the end of each block to know, what does it mean if there is no such feature set. Even if that would repeat something in the description, like gui.

And some mention that for the features mentioned this list is authoritative and if in some place there is another default, then said other place is wrong.